openshift/4.0/templates/Logging/docker-compose.yml

```diff
@@ -0,0 +1,35 @@
+version: "3.5"
```

Comment:
What's the purpose of this specific docker-compose?  Is it just for locally running it?

Reply:
@Fosol the docker-compose is just for locally running it, yes!

Reply:
Is there a way to test it locally?  A way to see the output?

Reply:
docker-compose up logging will build and run it locally
The log files will be in /tmp location. After logs are archieved and send to storage, all logs will be deleted off /tmp folder